compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -563,64 +613,77 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<bool>
       return false;
 
     // Check if this is bias
-    auto iw = get_input_weight_of_bias(node);
-    if (iw.first == nullptr || iw.second == nullptr)
-      return false;
-
-    auto input = loco::must_cast<luci::CircleNode *>(iw.first);
-    auto weight = loco::must_cast<luci::CircleNode *>(iw.second);
+    auto iwo_list = get_input_weight_output_of_bias(node);
 
-    if (granularity == QuantizationGranularity::ChannelWise)
+    for (auto iwo : iwo_list)
```

Comment:
From this line to the end, it seems change is not that much but the whole code shows diff.
Can you please use `// clang-format off` to reduce diff and `TODO: fix indentation` or an appropriate word?

Reply:
I've added clang-format off and on. It makes the PR much clearer. PTAL :)